src/profile-logic/marker-data.js

```diff
@@ -1394,7 +1399,7 @@ export function filterMarkerByDisplayLocation(
  * Compute the Screenshot image's thumbnail size.
  */
 export function computeScreenshotSize(
-  payload: ScreenshotPayload,
+  payload: { windowWidth: number, windowHeight: number },
```

Comment:
Looks like we had to do this because of the new type of `CompositorScreenshotWindowDestroyed`. This is also one of the reasons I would like to have new type for them in the future.

Reply:
yeah, this makes sure that the presence of these properties is done _before_ we call this function